src/test/java/commonmark_java_renderer/CommonmarkRendererTest.java

```diff
@@ -0,0 +1,15 @@
+package commonmark_java_renderer;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class CommonmarkRendererTest {
+
+	@Test
+	public void test_italics() {
```

Comment:
Rename to italics()
Using "test" is redundant because @Test.

Reply:
Okay!